### DISCUSSION TITLE
Add Rulers and Guidelines to the Editor Canvas

### DISCUSSION THREAD

**Discussion by submitaudio** [elementor/elementor#14022] [Feature Request]
Last week i was working in Wix for a client, i don't like Wix but one thing was really nice. Using Rulers and guides lines. 
Please is this possible?

**Comment by Muncamoo2000** [comment]
I also need Guidelines!!! or even just a ruler so make things uniform, especially when going from desktop editor to mobile editor. You can't use paste style because it messes with the desktop version creating more work. Using backend guidelines would be so helpful. Canva uses guidelines and even changes color when things are lined up. The same with Adobe InDesign (as a frame of reference.)
